index.html

```diff
@@ -592,6 +594,8 @@ <h2 class="bannerbox__title">2016 Election Countdown Clock</h2>
     </script>
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/i18n/jquery-ui-i18n.min.js"></script>  
```

Comment:
Same as above re: conditional lazy-loading and cutting down the JQUI build to just the bits we need.